wagtail/admin/templates/wagtailadmin/pages/bulk_actions/confirm_bulk_delete.html

```diff
@@ -0,0 +1,49 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags %}
+{% block titletag %}{% blocktrans count counter=pages|length %}Delete 1 page {% plural %}Delete {{ counter }} pages{% endblocktrans %}{% endblock %}
+
+{% block content %}
+    {% trans "Delete" as del_str %}
+    {% include "wagtailadmin/shared/header.html" with title=del_str icon="doc-empty-inverse" %}
+
+    <div class="nice-padding">
+        {% if pages %}
+        <p>{% trans "Are you sure you want to delete these pages?" %}</p>
+        <ul>
+            {% for page in pages %}
+            <li>
+                <a href="{% url 'wagtailadmin_pages:edit' page.page.id %}" target="_blank" rel="noopener noreferrer">{{ page.page.title }}</a>
+                {% if page.descendant_count %}
+                    <p>
+                    {% blocktrans count counter=page.descendant_count %}
+                    This will also delete one more subpage.
+                    {% plural %}
+                    This will also delete {{ counter }} more subpages.
+                    {% endblocktrans %}
+                    </p>
+                {% endif %}
+            </li>
+            {% endfor %}
+        </ul>
+        {% endif %}
+        {% if pages_with_no_access %}
+        <p>{% blocktrans count counter=pages_with_no_access|length %}The following page cannot be deleted{% plural %}The following pages cannot be deleted{% endblocktrans %}</p>
+        <ul>
+            {% for page in pages_with_no_access %}
+            <li>
+                <a href="{% url 'wagtailadmin_pages:edit' page.id %}" target="_blank" rel="noopener noreferrer">{{ page.title }}</a>
```

Comment:
Feels like we shouldn't offer the edit link if they don't have edit permissions (this is also what we do in the explore listing, see https://github.com/wagtail/wagtail/blob/main/wagtail/admin/templates/wagtailadmin/pages/listing/_page_title_explore.html)